explauto/environment/poppy/poppy_env.py

```diff
@@ -0,0 +1,60 @@
+from ... import Environment
+from ...utils import bounds_min_max
+
+
+class PoppyEnvironment(Environment):
+    """ Explauto environment for Poppy robots.
+
+        It can be used with all existing poppy creatures:
+            * Humanoid
```

Comment:
Is it not confusing as it is not the real name of the creature (it must be prefixed by Poppy) ?